### ISSUE TITLE
Investigate cold startup to homescreen creep up regression for 2/13+

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/fenix#8463]
On 2/6, we had a cold startup time of 1.16. Between 2/7 and 2/9, we had invalid runs https://github.com/mozilla-mobile/perf-frontend-issues/issues/65 so can't rely on that data. On 2/10, the time jumped to 1.20, which peaked on 2/11 at 1.25 and is approximately sustained. We should find out why this regression occurred so we can undo it.

We can get more data for investigation by repopulating FNPRMS https://github.com/mozilla-mobile/perf-frontend-issues/issues/68 (or doing something similar locally).

**Comment by pslawless** [comment]
Looking at the updated and repopulated FNPRMS numbers for the dates in question, it looks like the biggest regressions happened 2/10 and 2/11.

```
2020.02.06 	1.12194
2020.02.07 	1.0991899999999994
2020.02.08 	1.1034699999999995
2020.02.09 	1.1024199999999995
2020.02.10 	1.1157999999999997
2020.02.11 	1.2526200000000007
```

**Comment by mcomella** [comment]
We should use this bug to handle not the large regression on 2/10 but the slow creep up we're seeing from 2/13-2/24.

**Comment by emalysz** [comment]
I compared a 2/28 revision (f9b716286a899f7c5fe8cbd1dd58c354596cd202) with a 2/12 revision (cbc8d3aa45327f292e067b3e43aa877eaf845f73) and focused on files that I identified as affecting startup the most. The reason being that any changes to these files could cause the regression. The files were: FenixApplication, HomeActivity, GeckoProvider, and HomeFragment.

I was using GeckoBetaFenixBeta variant on a Pixel2. 

The most notable change I found was from https://github.com/mozilla-mobile/fenix/issues/7796, which was ~8.6ms.

Another change was to https://github.com/mozilla-mobile/fenix/commit/959e438ff3353db7c3913baa88018f066f2a2f55, but this only introduced 1-2 ms.
